### PR TITLE
Update user vars docs

### PR DESCRIPTION
**Pull request by therealsilence**
- Update Synapse user guide section on variables
  - Add line breaks for (internal) clarity
  - Revise user-assigned variables section for clarity, align with current data / analytical models
  - Minor cleanup / revision of overall content

### CODE REVIEW

File: docs/synapse/userguides/storm_adv_vars.rstorm [hunk 1]
```diff
@@ -232,204 +316,268 @@ The variable name must be specified first, followed by the equals sign and the v
...
 
+- Convert an epoch milliseconds integer into a human-readable date/time string using ``$lib.str.format()``:
 
+.. storm-cli:: $now=$lib.time.now() $time=$lib.time.format($now, '%Y/%m/%d %H:%M:%S') $lib.print($time)
 
-- If a file is tagged as part of a malicious code (malware) family, then also tag the file to indicate it is part of that malware's ecosystem:
+**Expressions**
 
+You can assign the value of a variable based on the computed value of an expression. To denote an expression
+to be evaluated, enclose the expression in parentheses preceded by a dollar sign (``$( <expression> )``).
```
Comment (Cisphyx) [inline docs/synapse/userguides/storm_adv_vars.rstorm:556]: you don't actually need the `$` to do this now. the only place where expressions require a `$` is in a filter (to differentiate from a compound filter)

Comment (Cisphyx) [reply docs/synapse/userguides/storm_adv_vars.rstorm:556]: also 'assign the value of a variable' doesn't sound quite right to me, 'assign a value to a variable' seems better

File: docs/synapse/userguides/storm_adv_vars.rstorm [hunk 2]
```diff
@@ -232,204 +316,268 @@ The variable name must be specified first, followed by the equals sign and the v
...
 
+.. storm-cli:: $x=5 $x=$($x + 1) $lib.print($x)
 
-.. NOTE:
 
-  The above query could be written as a **trigger** (:ref:`auto-triggers`) so that any time a ``#code.fam.<family>`` tag was applied to a file, the corresponding ``#cno.mal.<family>`` tag would be applied automatically.
+**Embedded Storm query**
 
-*Use a mathematical expression / "dollar expression" as a variable:*
+You can assign the value of a variable based on the output of a Storm query. To denote the Storm query to be
```
Comment (Cisphyx) [inline docs/synapse/userguides/storm_adv_vars.rstorm:565]: another 'assign the value of a variable'

File: docs/synapse/userguides/storm_adv_vars.rstorm [hunk 3]
```diff
@@ -232,204 +316,268 @@ The variable name must be specified first, followed by the equals sign and the v
...
+evaluated, enclose the query in curly braces (``{ <storm query> }``).
 
-- Use a mathematical expression to increment the variable ``$x``:
+- Assign an ``ou:org`` node's guid value to the variable ``$org`` by lifting the associated org node by its
+  ``:name`` propertys:
```
Comment (Cisphyx) [inline docs/synapse/userguides/storm_adv_vars.rstorm:569]: 'propertys'

**Review by reign-git — COMMENTED** [review]
Looks good!

**Pull request merged**